python/test/test_package.py

```diff
@@ -26,31 +26,31 @@ def test_schema_validity(self):
 
     def test_chalice_schema_generation(self):
         """
-        Validate that the schema generated by :func:`ga4gh.dos.schema.from_chalice_routes`
+        Validate that the schema generated by :func:`ga4gh.drs.schema.from_chalice_routes`
         is valid.
         """
-        from ga4gh.dos.schema import from_chalice_routes
+        from ga4gh.drs.schema import from_chalice_routes
         routes = {
             # Test a path that does not exist in the schema
             '/PathThatDoesNotExist': {'GET': None},
             # Test a valid path with a nonexistent method
-            '/ga4gh/dos/v1/databundles': {'MethodThatDoesNotExist': None},
+            '/ga4gh/drs/v1/bundles': {'MethodThatDoesNotExist': None},
             # Test a path with a different case than what is defined in the schema
-            '/GA4GH/DOS/V1/DATABUNDLES/{data_bundle_id}': {'GET': None},
+            '/GA4GH/DOS/V1/DATABUNDLES/{bundle_id}': {'GET': None},
             # Test multiple methods
-            '/ga4gh/dos/v1/dataobjects/{data_object_id}': {'GET': None,
+            '/ga4gh/drs/v1/objects/{object_id}': {'GET': None,
                                                            'PUT': None}
         }
-        schema = from_chalice_routes(routes, base_path='/ga4gh/dos/v1')
+        schema = from_chalice_routes(routes, base_path='/ga4gh/drs/v1')
         paths = schema['paths']
 
         self.assertNotIn('/PathThatDoesNotExist', paths.keys())
         # Test that base path is correctly stripped
-        self.assertNotIn('/ga4gh/dos/v1/databundles', paths.keys())
-        self.assertIn('/databundles/{data_bundle_id}', paths.keys())
-        self.assertNotIn('MethodThatDoesNotExist', paths['/databundles'].keys())
-        self.assertIn('get', paths['/dataobjects/{data_object_id}'].keys())
-        self.assertIn('put', paths['/dataobjects/{data_object_id}'].keys())
-        self.assertNotIn('/dataobjects', paths.keys())
+        self.assertNotIn('/ga4gh/drs/v1/bundles', paths.keys())
+        ############self.assertIn('/bundles/{bundle_id}', paths.keys())
```

Comment:
Not sure what happened here... this, and only this, line failed after my refactor.  It would be great to get some help looking into what's going on here.

Reply:
I think this test is expecting a path with the `basePath` stripped - try
```suggestion
        self.assertNotIn('/bundles', paths.keys())
```

Reply:
Thanks!  I'll commit your suggestion